Lib/test/test_os/test_windows.py

```diff
@@ -0,0 +1,640 @@
+import sys
+import unittest
+
+if sys.platform != "win32":
+    raise unittest.SkipTest("Win32 specific tests")
+
+import _winapi
+import fnmatch
+import mmap
+import os
+import shutil
+import signal
+import stat
+import subprocess
+import textwrap
+import time
+import uuid
+from test import support
+from test.support import import_helper
+from test.support import os_helper
+from .utils import create_file
+
+
+class Win32ErrorTests(unittest.TestCase):
```

Comment:
Originally, these tests were for WindowsError. But now OSError is raised on all platforms. These tests are no longer Windows specific, and most likely are duplicated in other parts.

Please check that all these functions are already tested with non-existing file (and `mkdir()` with an existing file). Then they can be deleted.